src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1559,6 +1536,10 @@ def create_ocsp_request(self, builder):
         return _OCSPRequest(self, ocsp_req)
 
     def _create_ocsp_basic_response(self, builder, private_key, algorithm):
+        if not isinstance(builder, ocsp.OCSPResponseBuilder):
```

Comment:
The reason we don't have this check here is because our OCSP response builder doesn't take a `backend` (since we're tentatively planning to abandon `backend` at some point due to lack of use). Since only we ever call `_create_ocsp_basic_response` (and it is a private method) we don't need to have the friendlier TypeError.

Reply:
Ok, will remove.